packages/@pollyjs/adapter-fetch/src/index.js

```diff
@@ -89,9 +89,7 @@ export default class FetchAdapter extends Adapter {
     const { absoluteUrl, response } = pollyRequest;
     const { statusCode } = response;
 
-    const responseBody =
```

Comment:
Moved this logic into the polly response object since we create Response instances with that object in a few places.